client/settings/survey-modal/index.js

```diff
@@ -137,8 +117,9 @@ const getOptionsArrayFromQuestions = ( surveyKey, surveyQuestion ) => {
 };
 
 const SurveyModal = ( { setOpenModal, surveyKey, surveyQuestion } ) => {
-	const { status } = useContext( WcPaySurveyContext );
```

Comment:
We're already consuming the context through `useContext( WcPaySurveyContext )`, so there's no need to have a new `useSurveySubmit` hook that does the same thing. I don't see `useSurveySubmit()` used anywhere else, so 🤷